packages/voting/src/voting.rs

```diff
@@ -26,6 +29,43 @@ pub enum Vote {
     Abstain,
 }
 
+#[derive(Serialize, Deserialize, Clone, Copy, PartialEq, JsonSchema, Debug)]
+pub struct MultipleChoiceVote {
+    // A vote indicates which option the user has selected.
+    pub option_id: u32,
```

Comment:
Think we want this to be a `u64` just to be pedantic so that we are sure it'll be able to index the vec of options without overflowing.

Reply:
but if we have to cast it to usize and usize is u32, isn't that an issue?